### PR TITLE
vscode: bundled ripgrep fixed

### PR DESCRIPTION
**Pull request by pshirshov**
This is a followup for https://github.com/NixOS/nixpkgs/pull/137912

This P/R just sets +x attribute on bundled ripgrep

### CODE REVIEW

File: pkgs/applications/editors/vscode/generic.nix [hunk 1]
```diff
@@ -121,6 +121,9 @@ let
         --replace "/usr/bin/pkexec" "/run/wrappers/bin/pkexec" \
         --replace "/bin/bash" "${bash}/bin/bash"
       rm -rf "$packed"
+
+      # this fixes bundled ripgrep
+      chmod +x resources/app/node_modules/vscode-ripgrep/bin/rg
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vscode/generic.nix:126]: Why are we not using the nixpkgs one?

**Comment by pshirshov** [comment]
Seems like it got broken again in 1.60.1. 

Now it fails saying "Failed to save 'configuration.nix': Unable to find pkexec or kdesudo.". In case I create `/usr/bin/pkexec" it says "Failed to save 'configuration.nix': User did not grant permission."

I can't find exact point in their sources which fails.

**Comment by pshirshov** [comment]
https://github.com/microsoft/vscode/blob/ee1655a82ebdfd38bf8792088a6602c69f7bbd94/src/vs/platform/native/electron-main/nativeHostMainService.ts#L514

**Comment by pshirshov** [comment]
Actually they delegate the job to https://github.com/jorangreef/sudo-prompt

File: pkgs/applications/editors/vscode/generic.nix (hunk 1 above)
Comment (pshirshov) [reply pkgs/applications/editors/vscode/generic.nix:126]: I'm not sure if the embedded one is not patched. Anyway it's already there.

**Comment by Artturin** [comment]
You're commenting in the wrong pr

**Comment by pshirshov** [comment]
Ah, my bad. I didn't notice that the P/R is not in unstable yet. Disregard the two messages above.

**Comment by Artturin** [comment]
https://nixpk.gs/pr-tracker.html?pr=137912

**Pull request merged**